style.css

```diff
@@ -83,3 +108,39 @@
 .article-container:nth-child(odd) {
   background-color: rgb(250, 249, 249);
 }
+
+.copyright-footer p {
+  border: 1px solid black;
+  margin: 1rem;
+  padding: 20px;
+}
+
+.copyright-footer {
+  margin-top: 70px;
+  margin-right: 15px;
+}
+
+#date {
+  text-align: right;
+  margin-right: 20px;
+}
+
+.contact-section {
+  /* display: none; */
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.